src/index.html

```diff
@@ -7,6 +7,378 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="top-header">
```

Comment:
what is top-header? do you have bottom header? think about class name

Reply:
Fixed.